x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/_suggestion_panel.scss

```diff
@@ -49,11 +49,6 @@
   height: 100%;
   width: 100%;
   pointer-events: none;
-  padding: $euiSizeS;
-
-  > .expExpressionRenderer {
-    position: static; // Let the progress indicator position itself against the button
```

Comment:
I agree that we shouldn't be targeting classes of components like this, but because `.expExpressionRenderer` sets a position of relative with no way to pass down a modifying class name or other prop, this needs to stay in here.

Reply:
@cchaos could you please expand why we need to keep this? All layouts of Lens charts are preserved, the look in this branch the same as on `master`.

> ...with no way to pass down a modifying class name or other prop...

Not sure what you mean. There is padding prop that Lens uses to change padding of `<ExpressionRenderer>`.

Reply:
The padding is fine, I'm speaking specifically of the `position: static`. The expression renderer has a loader that is is positioned absolutely against its wrapper. However, in Lens we  visually change what that wrapper is (via card / panel) and therefore it makes more sense to position the loader against Lens' wrapper. The only current way to handle the change of parent position, is by targeting this class. 

In an ideal world, I'd be able to pass a custom class name to the expression renderer that gets appended and then I can just target that class specifically. However, the component doesn't allow this and therefore I must target the class specifically (scoped to within the Lens wrapper).

Reply:
I don't think I completely understand. But, if it is needed, then lets add it, like:

```tsx
<RenderExpression expressionClassName={''} />
```

Reply:
Just pushed an update that adds `className` prop to `<ExpressionRenderer />`.

@cchaos let me know if I missed anything!

Reply:
Pushed a commit directly to fix the class target